auto_tests/tox_test.c

```diff
@@ -309,9 +309,9 @@ START_TEST(test_few_clients)
     uint8_t off = 1;
 
     while (1) {
-        tox_iteration(tox1);
-        tox_iteration(tox2);
-        tox_iteration(tox3);
+        tox_do_iteration(tox1);
+        tox_do_iteration(tox2);
+        tox_do_iteration(tox3);
 
```

Comment:
Oh for... it's been there for two weeks and *now* you complain about it? This is why I asked you to review each of the previously-11 commits separately. Why didn't you do so...?


```
  This is the 2nd commit message:

I pretty strongly believe tox_iteration needs to be renamed to a verb

There are several other noun functions in the API, but none of them *do* things.
I think even tox_do is better than tox_iteration.

tox_do_interval is one possibility, but I'm open to suggestions.
```

@iphy and I had a debate about this in the comments on the relevant commit, not that you were paying attention...